src/modules/Radio/Radio.tsx

```diff
@@ -0,0 +1,123 @@
+import React from "react";
+import uuid from "uuid/v4";
+import {
+  StandardProps,
+  WithStylesProps,
+  ThemeStyleSheetFactory
+} from "../../types";
+import { withStyles } from "../../base";
+
+interface IProps extends StandardProps<"input"> {
+  hasError?: boolean;
+  isDisabled?: boolean;
+  label: string;
+  name?: string;
+  selectedValue?: string;
+  value: string;
+  onValueSelect?: (value: string) => void;
```

Comment:
should use onChange of input

Reply:
As we discussed irl, I don't think that's what I want!